### PR TITLE
fix(translations): clean translations files

### PR DESCRIPTION
**Pull request by cbourgois**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | -
| License          | BSD 3-Clause

## Description

Clean translations to sync translations files between source (`fr_FR`) and targets: 
* Some translations are no more present in source translations files (`fr_FR`) and are still in target translations files.
* Some translations moved from one path to another and are still present in the previous path. It could be an issue depending on the translations loading (if translations were changed, they are not changed in old path).

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/hostedEmail/detail/translations/Messages_es_ES.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
   "hosted_email_detail_exchange_standard_1g": "Exchange 1 GB",
   "hosted_email_detail_exchange_basic_1g": "Exchange 1 GB",
   "hosted_email_detail_exchange_standard": "Exchange 25 GB",
-  "hosted_email_detail_hostedemail_individual": "Hosted 25 GB",
+  "hosted_email_detail_individual": "Hosted 25 GB",
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/hostedEmail/detail/translations/Messages_es_ES.json:35]: `hosted_email_detail_hostedemail_individual` removed, `hosted_email_detail_individual` moved in file.

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/voipLine/activation/translations/Messages_de_DE.json [hunk 2]
```diff
@@ -7,27 +7,19 @@
   "telefony_activation_total_error": "Beim Laden ist ein Fehler aufgetreten.",
   "telefony_activation_success_validation": "Votre demande a bien été prise en compte",
   "telefony_activation_an_error_ocurred": "Une erreur est survenue lors de la création de votre ligne",
-  "telefony_activation_shipping_address_label": "Choisissez l’adresse et le mode de livraison de votre téléphone",
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/voipLine/activation/translations/Messages_de_DE.json:10]: Translation key updated, so they are considered as removed : https://github.com/ovh/manager/commit/e610b48e0f37ef859a92f4bc6973466d6f12c310#diff-668ad20c7579fbee4ea12aa46bf0d1ff

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/translations/Messages_en_GB.json [hunk 3]
```diff
@@ -3,27 +3,7 @@
   "telephony_line_phone_accessories_load_error": "Oops! An error has occurred loading bill groups.",
   "telephony_line_phone_accessories_cancel_order": "Cancel accessories order",
   "telephony_line_phone_accessories_previous": "Previous step",
-  "telephony_line_phone_accessories_cisco_linksys_alim": "Alimentation pour Cisco Linksys 5V/2A",
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/translations/Messages_en_GB.json:6]: Removed : https://github.com/ovh/manager/commit/e610b48e0f37ef859a92f4bc6973466d6f12c310#diff-d65ce2565f3f519f74ad69378a81d1d2

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/translations/Messages_fi_FI.json [hunk 4]
```diff
@@ -6,12 +6,11 @@
   "telephony_line_phone_actions_line_programmable_keys": "Programmable keys",
   "telephony_line_phone_actions_line_phone_upgrade_firmware": "Firmware update",
   "telephony_line_phone_actions_line_phone_reboot": "Reboot",
-  "telephony_line_phone_actions_line_phone_order_plug_and_phone": "Commander un Plug & Phone",
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/translations/Messages_fi_FI.json:9]: Removed : https://github.com/ovh/manager/commit/e610b48e0f37ef859a92f4bc6973466d6f12c310#diff-754770ed58f43132b5382e06eb232a5fL9

File: packages/manager/apps/web/client/app/domains/translations/Messages_de_DE.json [hunk 5]
```diff
@@ -66,40 +66,6 @@
   "domains_configuration_dnssec_simple_activate_step1_question": "Möchten Sie wirklich DNSSEC für die Domain <span class=\"italic\">{{t0}}</span> aktivieren?",
   "domains_configuration_dnssec_simple_activate_success": "Ihre Anfrage zur Änderung des DNSSEC Zustands wurde erfolgreich durchgeführt.",
   "domains_configuration_dnssec_simple_activate_fail": "Bei der Anfrage für die Änderung des DNSSEC Zustands ist ein Fehler aufgetreten.",
-  "domains_newdnszone_order_title": "DNS Zone hinzufügen",
```
Comment (cbourgois) [inline packages/manager/apps/web/client/app/domains/translations/Messages_de_DE.json:69]: Moved in https://github.com/ovh/manager/blob/master/packages/manager/apps/web/client/app/dns-zone/new/translations/Messages_fr_FR.json
https://github.com/ovh/manager/commit/16b5fd75bc2622574a0dadd8b613031fe5b7ef77#diff-276ebfd9368c1fa4cee67b0980a9f6efL69

File: packages/manager/modules/hub/src/components/products/translations/Messages_de_DE.json [hunk 6]
```diff
@@ -3,6 +3,7 @@
   "manager_hub_products_CLOUD_PROJECT": "PCI Projekte",
   "manager_hub_products_CLOUD_DB_ENTERPRISE_CLUSTER": "Enterprise Cloud Database",
   "manager_hub_products_DBAAS_LOGS": "Logs Data Platform",
+  "manager_hub_products_DBAAS_QUEUE": "DbaaS Queue",
```
Comment (cbourgois) [inline packages/manager/modules/hub/src/components/products/translations/Messages_de_DE.json:6]: Lot of Updates (https://github.com/ovh/manager/commits/master/packages/manager/modules/hub/src/components/products/translations/Messages_fr_FR.json)

File: packages/manager/modules/sign-up/src/form/activity/translations/Messages_es_ES.json [hunk 7]
```diff
@@ -1,12 +1,5 @@
 {
   "sign_up_activity_field_vat": "Número de identificación",
-  "sign_up_activity_field_organisation": "Organización",
-  "sign_up_activity_field_corporationType": "Tipo de empresa",
-  "sign_up_activity_field_nationalIdentificationNumber": "NIF",
-  "sign_up_activity_field_companyNationalIdentificationNumber": "NIF",
-  "sign_up_activity_field_italian_sdi": "Código del destinatario para la factura electrónica (SDI)",
-  "sign_up_activity_field_vat_it": "Número de IVA",
```
Comment (cbourgois) [inline packages/manager/modules/sign-up/src/form/activity/translations/Messages_es_ES.json:8]: Removed : https://github.com/ovh/manager/commit/baa5a7d17d8c053b8291844884a9f8695f01f6af#diff-0efc62638fbd5c99c48244e6bad65fcaL3

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/hostedEmail/detail/translations/Messages_en_GB.json [hunk 8]
```diff
@@ -49,9 +49,8 @@
   "hosted_email_detail_client_delete_account_confirm": "Confirm deletion",
   "hosted_email_detail_client_delete_account_success": "Votre compte {{email}} est en cours de suppression.",
   "hosted_email_detail_client_delete_account_error": "Oups ! La suppression du compte a échoué.",
-  "hosted_email_detail_individual": "Hosted 25GB",
   "input_password_strength": "Password strength:",
   "input_password_danger": "Weak",
   "input_password_warning": "Average",
   "input_password_success": "High"
-}
+}
```
Comment (antleblanc) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/hostedEmail/detail/translations/Messages_en_GB.json:56]: I'm suggesting the following fix into our lint-staged configuration #2918
in order stop having `No newline at end of file` report.

**Comment by cbourgois** [comment]
Rebased on `develop` to use https://github.com/ovh/manager/pull/2918

**Pull request merged**